binderhub/health.py

```diff
@@ -119,15 +119,19 @@ def initialize(self, hub_url=None):
         return True
 
     @false_if_raises
+    @at_most_every(interval=10)
```

Comment:
```suggestion
    @at_most_every(interval=15)
```

(assuming since you said "15 seconds" in your comment